package.js

```diff
@@ -16,8 +16,9 @@ Package.onUse(function(api) {
   // Since JSX is only used when compiling and it isn't delivered to clients this isn't an issue, even for
   // those who aren't using React in their project.
 
-  api.use('jsx@0.1.3');
-  api.use('react@0.1.3', { weak: true });
+  api.use('jsx@0.1.4');
+  api.use('react@0.1.4', { weak: true });
+  api.use('react-meteor-data');
```

Comment:
I'm surprised that we need to declare react-meteor-data like this, since it's part of the react package. If the user's using react then react-meteor-data should be available - why isn't that the case? (It works in my setup).

By including it like this then it'll be compiled into the project for all clients, which isn't ideal (although it is very small, so it's not a major issue).

Assuming that it is necessary can you please add the version number required, otherwise I can't push to Atmosphere.